.github/workflows/component-library-tag-design.yml

```
@@ -0,0 +1,30 @@
+name: Component-Library-Tag-Design
+
+on:
+  pull_request:
+    types: [opened, ready_for_review, synchronize]
+    paths:
+      - 'frontend/packages/component-library-styles/**'
+      - 'frontend/packages/component-library/**'
+      - 'frontend/packages/fonts/**'
+
+jobs:
+  request:
+    name: Request review from design team on opened Component Library PRs
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
+
+    steps:
+      - name: Request Review
+        uses: actions/github-script@v7
```

Optional, since this is an official action, but probably good to have for consistency. [Pinning to the hash](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions) hardens the action to mitigate supply chain attacks for 3rd party actions. Dependabot detects and updates this still.

```suggestion
        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
```